restangular/restangular.d.ts

```diff
@@ -116,6 +116,9 @@ declare namespace restangular {
     withHttpConfig(httpConfig: angular.IRequestShortcutConfig): IElement;
     save(queryParams?: any, headers?: any): IPromise<any>;
     getRestangularUrl(): string;
+    route?: string;
+    id?: string;
```

Comment:
> Maybe using type of any would be
> better here to accommodate more endpoints?

If it can be an object then I guess `any` is fine, otherwise we can use string or number `id?: string | number;`